sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -105,13 +107,20 @@
     "mocha": "^6.2.2",
     "mocha-junit-reporter": "^1.18.0",
     "mocha-multi": "^1.1.3",
+    "nyc": "^14.0.0",
     "prettier": "^1.16.4",
     "rimraf": "^3.0.0",
     "rollup": "^1.16.3",
+    "rollup-plugin-shim": "^1.0.0",
     "rollup-plugin-sourcemaps": "^0.4.2",
     "rollup-plugin-terser": "^5.1.1",
     "rollup-plugin-visualizer": "^3.1.1",
     "sinon": "^7.1.0",
-    "typescript": "~3.6.4"
+    "source-map-support": "^0.5.9",
+    "typescript": "~3.6.4",
+    "@types/chai-as-promised": "^7.1.0",
+    "chai-as-promised": "^7.1.1",
```

Comment:
alphabetize plz

Reply:
why didn't rush do this for me

Reply:
It will do if you rely on `rush add` command.